x-pack/plugins/enterprise_search/server/routes/app_search/sample_response_search.test.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MockRouter, mockRequestHandler, mockDependencies } from '../../__mocks__';
+
+import { registerSampleResponseSearchRoute } from './sample_response_search';
```

Comment:
This route brings us back to a question we had in a number of other cases ... do we need to use *this* route, or can we use the regular search route. It's not clear to me whether or not this route does anything differently.

Reply:
I also wish I knew! I'll add this to https://github.com/elastic/app-search-team/issues/1631 as another endpoint we should consider though, I think I left it out when originally writing the ticket